### ISSUE TITLE
[FEATURE] AzureML Pipeline with Databricks integration

### ISSUE THREAD

**Issue by jreynolds01** [Microsoft/Recommenders#578]
### Description

Ye and I have done a bit of work with integrating AzureML Pipelines with ALS on spark to show how one can easily execute a notebook on Azure Databricks. We would like to integrate into the broader repository to demonstrate the value more publicly, but we are struggling a little with how to organize the resulting notebooks.

The user scenario: An individual wants to work from their local machine (or a pipeline agent), but wants to execute code on databricks. This code may be (re-)training a model (e.g. in response to new data), or this code may be (re-)scoring a model (e.g. in response to new data).

Right now, our end-to-end scenario for ALS on spark is purely in 1 notebook, and it is very rare that one would run all those steps with the same frequency.

### Expected behavior with the suggested feature

The proposal is to create a set of notebooks that covers the various steps required for a full end-to-end workflow, can be executed locally, and is cross-linked with the single notebook approach [als_movie_o16n.ipynb](https://github.com/Microsoft/Recommenders/blob/master/notebooks/05_operationalize/als_movie_o16n.ipynb)

The modularized notebook could look something like (names are not yet important):

```
notebooks/ALS-on-databricks-end-to-end-component-walkthrough.ipynb ## links to all other component notebooks:
notebooks/XX/create_azure_resources.ipynb  ## not sure where this could go - maybe rename 01_prepare_data to 01_prepare_resources??
notebooks/02_model/submit_ALS_databricks_train_job.ipynb
notebooks/05_operationalize/submit_ALS_databricks_rescore_job.ipynb
notebooks/06_operationalize/create_cosmosdb_lookup_job.ipynb
```

Then, we link from the full als_movie_o16n.ipynb notebook in a section that talks about how these different jobs may be done by different teams, or have different frequency requirements, so it makes sense to break them up.

### Other Comments
Posting this here for discussion on general thoughts on the approach, and then also specific thoughts on paths to place component pieces in.

**Comment by jreynolds01** [comment]
@miguel-ferreira  @dciborow  @yexing99

related to conversation going on on teams as well.

**Comment by miguelgfierro** [comment]
I think that makes sense

btw, poor @miguel-ferreira is getting weird messages about this repo :-) we are happy to get your contributions in case you are interested. However, most of the time, I think they refer to me: @miguelgfierro XDD

**Comment by jreynolds01** [comment]
gah! So sorry Miguel. I thought I had set the correct alias... Will be more careful!

**Comment by miguelgfierro** [comment]
lol

**Comment by gramhagen** [comment]
addressed in #682
